app/views/contexts/list_collector_summary_context.py

```diff
@@ -21,10 +21,9 @@ def get_list_summaries(self, current_location):
         visible_list_collector_blocks = self._schema.get_visible_list_blocks_for_section(
             section
         )
-        section_path = self._router.section_routing_path(
+        section_path_block_ids = self._router.section_routing_path(
```

Comment:
`section_path_block_ids` is a routing path and should be named `routing_path`. 

We should consider renaming the `section_routing_path` method to `routing_path` given that's how we refer to it.